### PR TITLE
Add Bedrock JCVD template

### PR DESCRIPTION
**Pull request by JGalego**
This PR adds a simple LangChain template that uses [Anthropic's Claude on Amazon Bedrock ⛰️](https://aws.amazon.com/bedrock/claude/) to behave like JCVD.

### CODE REVIEW

File: templates/bedrock-jcvd/jcvd_langserve.png
Comment (efriis) [inline templates/bedrock-jcvd/jcvd_langserve.png:1]: let's remove this for now. If you want the example in the readme, feel free to paste as text!

File: templates/bedrock-jcvd/bedrock_jcvd/chain.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "temperature": float(os.getenv("BEDROCK_JCVD_TEMPERATURE", "0.1")),
+    "top_p": float(os.getenv("BEDROCK_JCVD_TOP_P", "1")),
+    "top_k": int(os.getenv("BEDROCK_JCVD_TOP_K", "250")),
+    "max_tokens_to_sample": int(os.getenv("BEDROCK_JCVD_MAX_TOKENS_TO_SAMPLE", "300")),
+}
+
+# Full list of base model IDs is available at
+# https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids-arns.html
+_model_alts = {
+    "Claude 2.1": BedrockChat(
```
[the remark this answers is not in the document]
Comment (JGalego) [reply templates/bedrock-jcvd/bedrock_jcvd/chain.py]: is this a hard requirement? worked like a charm on the playground (better UX)
image

[the remark this answers is not in the document]
Comment (JGalego) [reply templates/bedrock-jcvd/bedrock_jcvd/chain.py]: any word on this ☝️?

**Comment by JGalego** [comment]
@efriis all requested changes have been implemented - anything else?

File: templates/bedrock-jcvd/bedrock_jcvd/chain.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (efriis) [reply templates/bedrock-jcvd/bedrock_jcvd/chain.py]: It technically works, and even the Python steering committee ended up [discussing this last year](https://discuss.python.org/t/non-identifier-names-of-kwargs-attributes-variables-etc/19293)! For the sake of keeping templates easier to understand, let's stick with valid identifiers here.

**Review by efriis — APPROVED** [review]

**Pull request merged**